.github/workflows/build.yml

```diff
@@ -72,4 +76,4 @@ jobs:
         with:
           path: ~/.cache/bazel
           key: bazel-${{ runner.os }}
-      - run: bazel build --cxxopt=-std=c++20 //...
+      - run: bazel build -c opt --cxxopt=-std=c++20 //...
```

Comment:
FYI we deliberately omitted opt to speed up builds. Internal CI has a ~7 min deadline and we are pretty close to exceeding that :/ Bazel is slower than CMake. Any thoughts on how else to speed up the build?

Reply:
No strong opinions :-)

Reply:
Let's remove opt then for now.